sord.gemspec

```diff
@@ -24,8 +24,9 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'yard'
   spec.add_dependency 'sorbet-runtime'
-  spec.add_dependency 'commander', '~> 4.5'
-  spec.add_dependency 'parlour', '~> 5.0'
+  spec.add_dependency 'commander', '~> 5.0'
+  spec.add_dependency "parser"
```

Comment:
We use parser explicitly now (rather than implicitly as part of Parlour) in order to negotiate the heredoc case.